include/rocksdb/db.h

```diff
@@ -99,6 +100,29 @@ struct Range {
   Range(const Slice& s, const Slice& l) : start(s), limit(l) { }
 };
 
+// <user key, seqeence number and entry type> tuple.
```

Comment:
Since FullKey references EntryType that's defined in table_properties.h which includes types.h, we'd need to move EntryType to types.h as well. Would that be fine?

Reply:
yes, I prefer moving EntryType into types.h regardless since now it's used for things unrelated to table properties.